nepal_compliance/overrides/salary_slip.py

```
@@ -0,0 +1,175 @@
+                    salary_slip_total -= sd.amount
+
+        total_loan_repayment = self.process_loan_repayments_for_bank_entry(salary_details) or 0
+        salary_slip_total -= total_loan_repayment
+
+        bank_entry = None
+        if salary_slip_total != 0:
+            remark = "withheld salaries" if for_withheld_salaries else "salaries"
+            bank_entry = self.set_accounting_entries_for_bank_entry(salary_slip_total, remark)
+
+            if for_withheld_salaries:
+                link_bank_entry_in_salary_withholdings(salary_details, bank_entry.name)
```

Ruff (0.12.2)

Undefined name `link_bank_entry_in_salary_withholdings`

(F821)